resources/css/app.css

```
@@ -14,6 +14,7 @@
 
 @plugin '@tailwindcss/typography';
 
+@custom-variant light (&:where(.light, .light *));
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)